### PR TITLE
improve runtime.MemStats and add -benchmem testing flag

### PR DESCRIPTION
**Pull request by dgryski**

### CODE REVIEW

**Comment by dgryski** [comment]
This addresses one of the issues preventing the `math/big` tests from running. ( https://github.com/tinygo-org/tinygo/issues/890 )

File: src/runtime/gc_conservative.go [hunk 1]
```diff
@@ -55,7 +55,9 @@ var (
 	metadataStart unsafe.Pointer // pointer to the start of the heap metadata
 	nextAlloc     gcBlock        // the next block that should be tried by the allocator
 	endBlock      gcBlock        // the block just past the end of the available space
-	gcTotalAlloc  uint64         // for runtime.MemStats
+	gcTotalAlloc  uint64         // total number of bytes allocated
+	gcMallocs     uint64         // total number of alloations
```
Comment (deadprogram) [inline src/runtime/gc_conservative.go]: Misspelling in comment. :smiley_cat:

**Comment by deadprogram** [comment]
Looks good to me! @aykevl anything to say before I merge?

**Review by deadprogram — APPROVED** [review]
Looked over again and seems good. Thank you @dgryski

**Pull request merged**